### ISSUE TITLE
PostScript backend

### ISSUE THREAD

**Issue by pcwalton** [servo/pathfinder#200]
It would be nice to have a PostScript *backend*, so that you can construct Pathfinder scenes and then print them. (This has potential Firefox applications.)

**Comment by s3bk** [comment]
Why PostScript as opposed to PDF?

**Comment by pcwalton** [comment]
Because as I understand it printers take PostScript.

**Comment by s3bk** [comment]
I wrote a small [PS exporter](https://github.com/s3bk/pathfinder/blob/ps_export/renderer/src/scene.rs#L202).
I wonder if the exporters should be moved into their own crate and be added via SceneExt traits?

**Comment by pcwalton** [comment]
I think that would be a good idea, yeah.

**Comment by s3bk** [comment]
SceneProxy depends on SVG export (and ideally it should also allow PDF and PS). This causes a cyclic crate dependency between renderer and export (the new crate with SVG, PDF and PS export).

**Comment by pcwalton** [comment]
Yeah, we can just get rid of the `SceneProxy` `Debug` implementation.
